core/src/main/java/org/elasticsearch/http/netty/NettyHttpChannel.java

```diff
@@ -85,6 +88,9 @@ public BytesStreamOutput newBytesOutput() {
 
     @Override
     public void sendResponse(RestResponse response) {
+        if (closed.get()) {
```

Comment:
Ah, that's a leftover. I remove it as the logic is now contained in `ResourceHandlingHttpChannel`.